makefiles/tests/tests.inc.mk

```diff
@@ -74,7 +74,10 @@ test-with-config/check-config:
 # this target only makes sense if an ELFFILE is actually created, thus guard by
 # RIOTNOLINK="".
 ifeq (,$(RIOTNOLINK))
-test-input-hash: $(TESTS) $(TESTS_WITH_CONFIG) $(TESTS_AS_ROOT) $(ELFFILE) $(TEST_EXTRA_FILES)
+  ifeq (,$(FLASHFILE))
+    $(error "FLASHFILE is not set!")
+  endif
```

Comment:
I guess this check isn't needed here because this check is already done here https://github.com/RIOT-OS/RIOT/blob/55b57e1a75a1e447086161012bee9ff499474d96/Makefile.include#L685-L687 before `makefile/tests/test.inc.mk` is included. Right?

Reply:
BTW, using the undefined `FLASHFILE` variable in the error message doesn't make sense :wink:

Reply:
The intention was probably to use `$(BOARD)`